pkg/apis/samples/v1alpha1/samplesource_types.go

```diff
@@ -45,21 +44,24 @@ type SampleSource struct {
 }
 
 // GetGroupVersionKind returns the GroupVersionKind.
-func (s *SampleSource) GetGroupVersionKind() schema.GroupVersionKind {
+func (*SampleSource) GetGroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind("SampleSource")
 }
 
-// Check that SampleSource is a runtime.Object.
-var _ runtime.Object = (*SampleSource)(nil)
-
-// Check that SampleSource can be validated and can be defaulted.
-var _ resourcesemantics.GenericCRD = (*SampleSource)(nil)
-
-// Check that we can create OwnerReferences to a SampleSource.
-var _ kmeta.OwnerRefable = (*SampleSource)(nil)
```

Comment:
Why did we drop the `kmeta.OwnerRefable` comment?

Reply:
Oh sorry I am completely forget to leave that. plz see next commit.